src/components/MoveButton.vue

```diff
@@ -102,7 +102,7 @@ export default class MoveButton extends Vue {
         const keys: GaiaHex[] = [...highlighted.keys()];
         this.$store.commit("gaiaViewer/highlightHexes", new Map([...keys.map(key => [key, null])] as any));
       });
-    } else if ( this.button.command.split(' ')[0] === "faction") {
+    } else if ( this.button.command.split(' ')[0] === Command.ChooseFaction) {
```

Comment:
should be changed to `if(this.button.modal){`.

That way, we can also display modals for other things (like a warning when the player does something stupid, like declining 1 leech, or disacarding power tokens in zone 3, or discarding brainstone)